docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -41,30 +41,34 @@ can be configured with the `spec.options.device_trust_mode` option and
 applies to the resources in its `allow` rules. It
 works similarly to [`require_session_mfa`](../guides/per-session-mfa.mdx).
 
-<Admonition type="tip" title="v13.3.6+">
-  Teleport version 13.3.6 and above has the preset `require-trusted-device` role.
-  Make sure you update the "allow" rules in the role according to your requirements.
+<Admonition type="warning" title="require-trusted-device role">
+  The preset `require-trusted-device` role does not enforce
+  [Apps](#app-access-support) or [Desktops](#desktop-access-support).
```

Comment:
Should this say "does not enforce the use of a trusted device for apps for desktops" ?

Reply:
Done.